app/controller.go

```diff
@@ -66,6 +67,12 @@ func startController(c *cli.Context) error {
 		return errors.New("volume name is required")
 	}
 	name := c.Args()[0]
+	replicationFactor, _ := strconv.ParseInt(os.Getenv("REPLICATION_FACTOR"), 10, 32)
```

Comment:
this code change should be enough if we set c.replicaCount to replicationFactor here itself. No need of other changes given in this PR.